doc/dev/ref-manual.md

```diff
@@ -91,14 +91,9 @@ Any memory that needs to be revocable must have one revocation tag per 32 bits.
 
 ### Capability tags
 
-All capability tags live in SRAM.
-All SRAM that is allocated for code and data will have corresponding capability tags.
-Any data stored to HyperRAM and flash are not expected to be tagged.
-Since capability tags are out of band information and do not need to be memory mapped, we can store these within the error correction bits that are available on the FPGA's memory.
-
+All of SRAM and 1 MiB of HyperRAM have corresponding capability tags.
+We envision code and read-only data to live in HyperRAM, while the stack and the heap live in SRAM.
 We envision that code can live in HyperRAM with an instruction cache for improved performance.
```

Comment:
Two sentences starting with "We envision".

Reply:
Thanks, removed the second one.